src/tokenizer.jl

```diff
@@ -1,9 +1,33 @@
-##############################################################################
```

Comment:
I just realised that keeping them might be a good idea as they give a proper separation of code-blocks. If creates more visibility of code sections for users who are there for a quick view of code.

Reply:
You mean in a visual sense only? It may be better not to follow a different set of the convention for just this package though in my opinion.

Reply:
Yeah in that sense. If u see, different features are separated by that along with related function. So, while working on the docs, just found the stuff a little less intuitive.

Reply:
Should I mark this resolved? @aquatiko

Reply:
Yeah sure, I also removed the redundant ones from my part of PR keeping only the ones with some critical info and TODO marked.

Reply:
Yup, same for this PR.